.github/workflows/deploy-catalog.yaml

```diff
@@ -0,0 +1,56 @@
+name: Deploy catalog to ECR
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/deploy-catalog.yaml'
+      - 'catalog/**'
+      - 'shared/**'
+
+jobs:
+  deploy-catalog-ecr:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: catalog
+    env:
+      NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
No idea if this is useful - I just copied it from js-ci.yml

Reply:
I think it's not